src/new/sendTransaction.ts

```diff
@@ -6,18 +6,43 @@
  */
 
 import EventEmitter from 'events';
-import { MIN_POLLING_INTERVAL, SELECT_OUTPUTS_TIMEOUT, HATHOR_TOKEN_CONFIG } from '../constants';
-import transaction from '../transaction';
-import helpers from '../utils/helpers';
+import { SELECT_OUTPUTS_TIMEOUT, HATHOR_TOKEN_CONFIG } from '../constants';
+import transactionUtils from '../utils/transaction';
 import txApi from '../api/txApi';
 import { WalletError, SendTxError } from '../errors';
 import { ErrorMessages } from '../errorMessages';
-import wallet from '../wallet';
-import oldHelpers from '../helpers';
-import storage from '../storage';
+import helpers from '../utils/helpers';
 import MineTransaction from '../wallet/mineTransaction';
 import Address from '../models/address';
 import { OutputType } from '../wallet/types';
+import { IStorage, IDataTx, IFillTxOptions } from '../types';
+import Transaction from '../models/transaction';
+
+export interface ISendInput {
+  txId: string,
+  index: number,
+};
+
+export interface ISendDataOutput {
+  type: OutputType.DATA,
+  data: Buffer,
+  value?: number,
+  token?: string,
+}
+
+export function isDataOutput(output: ISendOutput): output is ISendDataOutput {
```

Comment:
I tried to use the `IDataXXX` types of `src/types.ts` here but the `SendTransaction` facade requires another format of data which comes from user input.
Since I do not want to change the facade arguments I created these `ISendXXX` types which are used only here.

The `isDataOutput` which returns `output is ISendDataOutput` does not do anything, it only provides a way to signal the ts compiler which of the of the possible types (`ISendDataOutput` and `ISendTokenOutput`) is the output.